bmorph/util/mizuroute_utils.py

```diff
@@ -287,26 +392,30 @@ def map_ref_sites(routed: xr.Dataset, gauge_reference: xr.Dataset,
                     gauge_sites=None, route_var = 'IRFroutedRunoff',
                     fill_method='kldiv'):
     """
-    map_ref_sites
-        assigns segs within routed boolean 'is_gauge' idnetifiers and
-        what each seg's upstream and downstream reference seg designations
-        are
-    ----
-    routed: xr.Dataset
-    gauge_reference: xr.Dataset
-    gauge_sites = None
-        if None, gauge_sites will be taken as all those listed in
-        gauge_reference
-    route_var = 'IRFroutedRunoff'
-        variable name of flows used for fill_method purposes within routed
+    Assigns segs within routed boolean 'is_gauge' idnetifiers and
```

Comment:
Make sure it's clear that `routed` is the input. Just enclose with ``. Also "idnetifiers" -> "identifiers"